### ISSUE TITLE
Horus X10(S)-ISRM und 1.5.0

### ISSUE THREAD

**Issue by Wolepo** [FrSkyRC/ETHOS-Feedback-Community#3271]
Hello, will a version for the Horus X10(S)-ISRM for Ethos 1.5.0 be delivered later? 
Thank you
Wolfgang

**Comment by digitalonkel** [comment]
Thanks Wolfgang, i am looking forward to getting this variant as well.
Regards Rainer

**Comment by mawzthefinn** [comment]
Yes, it's been mentioned elsewhere that there will be X10 ISRM support by release time, the nightlies are Express and X12S ISRM only for now.

**Comment by Wolepo** [comment]
Thank you for the comforting words.
